tests/baselines/reference/api/typescript.d.ts

```diff
@@ -5170,6 +5195,12 @@ declare namespace ts {
     function isJSDocImplementsTag(node: Node): node is JSDocImplementsTag;
     function isJSDocSatisfiesTag(node: Node): node is JSDocSatisfiesTag;
     function isJSDocThrowsTag(node: Node): node is JSDocThrowsTag;
+    function isBinaryOperatorToken(node: Node): node is BinaryOperatorToken;
+    const isQuestionOrExclamationToken: (node: Node) => node is QuestionToken | ExclamationToken;
```

Comment:
These declarations are misbehaving in d.ts emit (ordering changes randomly, weird); I will rewrite them as functions to match all of the others.